Sming/SmingCore/Debug.cpp

```diff
@@ -18,20 +18,21 @@ DebugClass::~DebugClass()
 void DebugClass::initCommand()
 {
 #if ENABLE_CMD_EXECUTOR
-	commandHandler.registerCommand(CommandDelegate("debug", "New debug in development", "Debug",
+	auto debug = F("Debug");
```

Comment:
Why you add a separate variable for the debug text that is used only once? Maybe for consistency it would be better to write `CommandDelegate(debug, F("New debug in development"), F("Debug"),`

Reply:
Sorry, that's an error - I mistook both strings to be the same, but case is different.